lib/galaxy/util/xml_macros.py

```diff
@@ -15,36 +15,29 @@ def load_with_references(path):
     tree = raw_xml_tree(path)
     root = tree.getroot()
 
-    macro_paths = _import_macros(root, path)
-
-    # temporarily remove the children of the macros node
-    # and create a copy. this is done because this allows
-    # to iterate over all expand nodes of the tree
-    # that are not included in the macros node
     macros_el = _macros_el(root)
-    if macros_el is not None:
-        macros_copy = copy(macros_el)
-        macros_el.clear()
-    else:
-        macros_copy = None
+    if macros_el is None:
+        return tree, []
+
+    macros = {}
```

Comment:
```suggestion
    macros: Dict[str, XmlMacroDef] = {}
```
I think that'd be the right type annotation that would make mypy happy.